content/asset-tracking/example-apps.textile

```diff
@@ -63,6 +63,43 @@ blang[android].
     <img src="/images/screenshots/asset-tracking/android/example-apps-running.png" style="width: 100%" alt="Example apps running">
   </a>
 
+blang[swift].
+  h2(#using). Using the example apps
+
+  1. Clone the Ably Asset Tracking SDK Swift "GitHub repo":https://github.com/ably/ably-asset-tracking-swift.
+  2. Open @AblyAssetTracking.xcworkspace@ in XCode. "This workspace includes both the SDKs and the example apps":https://github.com/ably/ably-asset-tracking-swift#package-structure.
```

Comment:
Xcode is spelt `Xcode`, with lowercase `c`

Reply:
Good catch!